webapp/src/components/common/FormTable/Table.tsx

```diff
@@ -36,7 +36,7 @@ function Table(props: TableProps) {
                 ? longestHeaderLength
                 : headerLength;
             },
-            3 // To force minimum size
+            20 // To force minimum size
```

Comment:
Are you sure you are not changing others tables?

Reply:
I agree maybe its too much for other tables, but we need to be able to handle specific cases with long zone names 

![image](https://github.com/AntaresSimulatorTeam/AntaREST/assets/33469289/8ace4228-2506-4455-a870-e1a4da5a10a7)